### DISCUSSION TITLE
No idea what to do with Dependabot alerts

### DISCUSSION THREAD

**Discussion by avi12** [community/community#63374] [Code Security]
### Select Topic Area

Product Feedback

### Body

Recently I got a Dependabot security report over email and [this is one of the entries](https://github.com/avi12/skillshare-player-control/security/dependabot)
However, I don't directly use any of these dependencies
Moreover, even if I search within the [lockfile](https://github.com/avi12/skillshare-player-control/blob/main/pnpm-lock.yaml), I cannot see any dependency that's directly listed in package.json's `dependencies`/`devDependencies`, i.e. I have no idea what module version(s) I need to bump
Currently, the only way I know to solve these alerts is by manually searching in `node_modules` for scripts that import the affected dependencies and see what directories import them

**Comment by adityarizkyramadhan** [comment]
I am also having this problem, can anyone help?

**Comment by arjunkumarbose** [comment]
If you think you don’t want to use it, you can disable it as well
Check here,
https://docs.github.com/en/code-security/dependabot/dependabot-alerts/configuring-dependabot-alerts

**Reply by avi12** [reply]
I do want it. I just need clearer alerts so that I know what module(s) I need to upgrade in `package.json`

**Reply by arjunkumarbose** [reply]
Then you should check out the ways to configure it so that the version controls of the module can be managed; as if it varies based on your projects and packages.
In my case, I went threw this article to understand,
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

**Reply by arjunkumarbose** [reply]
I had to troubleshoot in some cases, for that:
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/troubleshooting-the-detection-of-vulnerable-dependencies
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/troubleshooting-dependabot-errors
